pkg/controller/master/nodedrain/nodedrain_controller.go

```diff
@@ -0,0 +1,293 @@
+package nodedrain
+
+import (
+	"context"
+	"fmt"
+
+	longhornv1 "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta1"
+	ctlcorev1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/rest"
+	v1 "kubevirt.io/api/core/v1"
```

Comment:
prefer to use same package alias name like other package:
```go
kubevirtv1 "kubevirt.io/api/core/v1"
ctlkubevirtv1 "github.com/harvester/harvester/pkg/generated/controllers/kubevirt.io/v1"
ctllonghornv1 "github.com/harvester/harvester/pkg/generated/controllers/longhorn.io/v1beta1"
```

Reply:
👍